Lua/Lua.sublime-syntax

```diff
@@ -82,6 +87,20 @@ contexts:
     - include: end
     - include: statements
 
+  shebang:
+    - match: ^\#!
+      scope: punctuation.definition.comment.lua
+      set: shebang-body
+    - match: ''
+      pop: 1
```

Comment:
Not a big deal, but I'd vote to not mix pop styles a document. Either `pop: true` for all or `pop: 1`. We might switch to latter one once syntax version 2 is applied.

Reply:
There is no other Lua PR so no conflict! Let's change them to all `1`s.